dfget/core/uploader/peer_server.go

```diff
@@ -482,7 +488,9 @@ func (ps *peerServer) deleteExpiredFile(path string, info os.FileInfo,
 		if ok && !task.finished {
 			return false
 		}
-		if time.Now().Sub(info.ModTime()) > expireTime {
+
+		// if the last access time is expireTime ago
+		if time.Since(task.accessTime) > expireTime {
```

Comment:
use the bigger one of `access time` and `modify time` to check whether the task is expired

Reply:
Done.